frontend/src/components/widgets/floating-menus/MenuList.vue

```diff
@@ -240,6 +241,78 @@ const MenuList = defineComponent({
 
 			return this.open;
 		},
+
```

Comment:
> Also, we are currently crashing when we change blend modes with the arrow keys.

Can't replicate this

Reply:
On https://cabeebb4.graphite-master.pages.dev

https://files.keavon.com/-/GenuineTatteredBedbug/capture_2_.mp4

Reply:
> This change will also mean you don't need your highlighted: this.activeEntry as MenuListEntry | undefined, in the data section of the Vue component anymore, since the highlighted value is the real value not a temporary one.

This breaks the menu bar entries as well as doing things like triggering a popup with the design mode dropdown when the user is still navigating hasn't selected one. This also makes it impossible to restore the old option when the esc key is pressed.

Reply:
@Keavon Crash doesn't happen on https://d3c81f1b.graphite-master.pages.dev/

Reply:
Hmm, true, that is a complication. But for DropdownInput menus, that's still a relatively important bit of behavior for the UX (especially with blend modes, people will be annoyed it's missing, but all dropdown menus too). We'll probably need to store the state that <kbd>Esc</kbd> restores to. And it'll be necessary to find a solution for disabling the behavior with menu bars (maybe make it opt-in for the dropdown and font widgets). I think we can live with the design mode input triggering coming soon dialogs each time, that's unlikely to be encountered by a user and it'll eventually behave as expected even that feature is implemented.

Reply:
Crash seems to be resolved on that build, yes.

Reply:
Perhaps a `interactive` prop that if true means that emit is called?

Reply:
Sounds like a good plan 👍

Reply:
Done